### ISSUE TITLE
VLC doesn't work, other players are fine

### ISSUE THREAD

**Issue by rand0MPrecisi0n** [trizen/youtube-viewer#211]
Hello and thanks for your time again. 
Lately I've wanted to make everything work with VLC, but I wasn't able to make youtube-viewer do it. 
I've tried a whole lot of stuff but VLC keeps opening and instantly closing. Mpv and mplayer work fine. 

Here's the error message it gives

> VLC media player 3.0.2 Vetinari (revision 3.0.2-0-gd7b653cf14)
> qt5ct: using qt5ct plugin
> qt5ct: D-Bus global menu: no
> qt5ct: D-Bus system tray: no
> QObject::~QObject: Timers cannot be stopped from another thread

So I checked qt5ct's version and it is the newest available in my repo (0.34-1build2).
After that I removed it and tried again, which gave me:

> VLC media player 3.0.2 Vetinari (revision 3.0.2-0-gd7b653cf14)
> QObject::~QObject: Timers cannot be stopped from another thread

Re-installed it, and tried to remove youtube-viewer.conf, but got the same error
Then I tried the solution in issue #96, running the following command:

> vlc $(youtube-viewer --no-interactive --no-dash --extract '*VIDEO*' 'http://www.youtube.com/watch?v=Cvu1HnAVuww')`

This worked and gave me this message:

> `main libvlc: Running vlc with the default interface. Use 'cvlc' to use vlc without interface.
> qt5ct: using qt5ct plugin
> qt5ct: D-Bus global menu: no
> qt5ct: D-Bus system tray: no

It still didn't work from inside youtube-viewer, so I tried to remove --play-and-exit from youtube-viewer.conf vlc section, which made VLC give me this error:

> Your input can't be opened:
> VLC is unable to open the MRL 'https://r1---sn-bxpbiucg51pa-btoe.googlevideo.com/videoplayback?sparams=aitags%2Cclen%2Cdur%2Cei%2Cgir%2Cid%2Cinitcwndbps%2Cip%2Cipbits%2Citag%2Ckeepalive%2Clmt%2Cmime%2Cmm%2Cmn%2Cms%2Cmv%2Cpl%2Crequiressl%2Csource%2Cexpire&signature=37B76A81FD15A020181335AA2CC74833A5A3A95F.8B6E22E493D042C971E7AFC332B64F4038D90D8C&itag=248&c=WEB&id=o-ADrruIlauS8a_5zTcgiaNcoblJZBGG3S59J1Gdp4jaWH&mn=sn-bxpbiucg51pa-btoe%2Csn-bg0e7n7k&mm=31%2C29&ip=170.82.79.1&ms=au%2Crdu&ei=mLtXW5-aJInq-QPJrI1I&mv=m&pl=24&ipbits=0&keepalive=yes&aitags=133%2C134%2C135%2C136%2C137%2C160%2C242%2C243%2C244%2C247%2C248%2C278&dur=841.640&fvip=5&source=youtube&lmt=1532457316024809&clen=273412287&expire=1532497912&key=yt6&gir=yes&mime=video%2Fwebm&mt=1532476235&requiressl=yes&initcwndbps=785000'. Check the log for details.

Wasn't able to make VLC to log stuff, so I added --debug to youtube-viewer's options and it gave me this:

> -> Resolution: 1080
> -> Video itag: 96
> -> Video type: video/mp4
> -> Command: vlc --quiet --no-video-title-show --input-title-format Teste\ Ao\ Vivo\ Monitor\ AOC\ Agon\ AG322FCX\ Com\ Freesync\ e\ 120Hz\ no\ Xbox\ One\ X https\:\/\/manifest\.googlevideo\.com\/api\/manifest\/hls_playlist\/id\/yk7hNK4tXcg\.1\/itag\/96\/source\/yt_live_broadcast\/requiressl\/yes\/ratebypass\/yes\/live\/1\/cmbypass\/yes\/goi\/160\/sgoap\/gir\%3Dyes\%3Bitag\%3D140\/sgovp\/gir\%3Dyes\%3Bitag\%3D137\/hls_chunk_host\/r2\-\-\-sn\-bxpbiucg51pa\-btoe\.googlevideo\.com\/playlist_type\/DVR\/gcr\/br\/ei\/x71XW\-jUKc_9\-QOurpygBQ\/initcwndbps\/5580\/mm\/32\/mn\/sn\-bxpbiucg51pa\-btoe\/ms\/lv\/mv\/m\/pl\/24\/dover\/10\/keepalive\/yes\/mt\/1532476758\/disable_polymer\/true\/ip\/170\.82\.79\.1\/ipbits\/0\/expire\/1532498471\/sparams\/ip\,ipbits\,expire\,id\,itag\,source\,requiressl\,ratebypass\,live\,cmbypass\,goi\,sgoap\,sgovp\,hls_chunk_host\,playlist_type\,gcr\,ei\,initcwndbps\,mm\,mn\,ms\,mv\,pl\/signature\/64B6DF4C7C96DDC3B8659ED9FA2DB73A8EB57842\.26441F85086E71F402D1ACB52638EBAA3DAFD9DB\/key\/dg_yt0\/playlist\/index\.m3u8
> VLC media player 3.0.2 Vetinari (revision 3.0.2-0-gd7b653cf14)
> qt5ct: using qt5ct plugin
> qt5ct: D-Bus global menu: no
> qt5ct: D-Bus system tray: no
> QObject::~QObject: Timers cannot be stopped from another thread

What should I do?

**Comment by rand0MPrecisi0n** [comment]
I did that and got the following error:

`Your input can't be opened:
VLC is unable to open the MRL 'https://www.youtube.com/watch?v=QtXby3twMmI'. Check the log for details.`

The terminal says:

`VLC media player 3.0.3 Vetinari (revision 3.0.3-1-0-gc2bb759264)
[0000564e31b74630] main libvlc: Running vlc with the default interface. Use 'cvlc' to use vlc without interface.
qt5ct: using qt5ct plugin
qt5ct: D-Bus global menu: no
qt5ct: D-Bus system tray: no
[00007f6224001600] access stream error: HTTP connection failure`

When I run the same video with `vlc URL` it works and gives me this (I cancelled with CTRL-C in the end)

![vlc](https://user-images.githubusercontent.com/22467805/44615155-f34e4d80-a808-11e8-9fff-230b126a6422.png)

As I learned more about I started to enjoy it, but I'm posting regardless, for the sake of continuity. Feel free to forget this problem altogether, and thanks for your time again :+1:
